### PR TITLE
[v0.0.11] Prepare release: config.yaml, Readmes, docs, translations

### PR DESCRIPTION
**Pull request by raphmur**
Please commit here on Changelog & Readmes, ...
Will close this PR at the end

### CODE REVIEW

File: tesla_ble_mqtt/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## 0.0.11
+
+### Changed
+
+- No need to provide MAC addresses anymore for presence detection
```
Comment (baylanger) [inline tesla_ble_mqtt/CHANGELOG.md:7]: This is true with the Expect ble-presence but not with the current sh script. At least not as of now...

There are a few approaches that we can use to do it. The simplest one would be to send to the addon's log the bluetoothctl output until we matches all VIN (BLE Local Name) part of the list. The end user would copy / paste the MAC address. Doesn't require the end user to download an app on his smartphone and figure out where to look.

But let me think about this ... I'll get back to you.

File: tesla_ble_mqtt/config.yaml [hunk 2]
```diff
@@ -17,33 +17,25 @@ map:
...
   send_cmd_retry_delay: "5"
 schema:
-  vin1: str?
-  vin2: str?
-  vin3: str?
-  presence_detection: bool
-  ble_mac1: str?
-  ble_mac2: str?
-  ble_mac3: str?
+  vin_list: "match(^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$)"
```
Comment (baylanger) [inline tesla_ble_mqtt/config.yaml:30]: Still need to test these regex.

File: tesla_ble_mqtt/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
+- Improved presence detection
+- Improved logging (bashio timestamps, verbosity, colors)
+- Added "debug" entity which sends only one charge amps command: https://github.com/tesla-local-control/tesla_ble_mqtt_core/issues/19
+- Clarified issues with BLE device overheating causing performance issues: https://github.com/tesla-local-control/tesla-local-control-addon/issues/27
```
Comment (baylanger) [inline tesla_ble_mqtt/CHANGELOG.md:12]: We can have this but long term I think this should be in a Troubleshooting section or readme ?!

File: tesla_ble_mqtt/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
 
+## 0.0.11
+
+### Changed
+
+- No need to provide MAC addresses anymore for presence detection
+- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
+- Improved presence detection
+- Improved logging (bashio timestamps, verbosity, colors)
+- Added "debug" entity which sends only one charge amps command: https://github.com/tesla-local-control/tesla_ble_mqtt_core/issues/19
```
Comment (baylanger) [inline tesla_ble_mqtt/CHANGELOG.md:11]: Is this still valid? I thought we were sending this command twice?! Or is the "debut" entity sending just 1 command to see if it works?!

Comment (raphmur) [reply tesla_ble_mqtt/CHANGELOG.md:11]: debug entity is to send only once.
So we have the "passe partout" which sends twice if below 5 and the "debug" which sends once only.
For amps below 5a, the debug will not work immediately and will need to be called twice from HA

File: tesla_ble_mqtt/CHANGELOG.md (hunk 3 above)
Comment (raphmur) [reply tesla_ble_mqtt/CHANGELOG.md:12]: See https://github.com/tesla-local-control/tesla-local-control-addon/pull/62 --> Will merge to this PR.

File: tesla_ble_mqtt/CHANGELOG.md (hunk 4 above)
Comment (baylanger) [reply tesla_ble_mqtt/CHANGELOG.md:11]: Why is it called "debug"?  Are we troubleshooting this issue?  I feel confused by the word debug.

File: tesla_ble_mqtt/config.yaml (hunk 2 above)
Comment (baylanger) [reply tesla_ble_mqtt/config.yaml:30]: It works, so lets keep the regex.

**Pull request closed without merging**